service/geopmdpy_test/TestActiveSessions.py

```diff
@@ -339,16 +192,21 @@ def test_creation_bad_session_perms(self):
         # first time in __init__() it is called with a string to determine the directory
         # second time in _load_session_file() it is called with a file descriptor to determine the file
         # based on the data type we use a different mock for the os.stat()
-        side_effect = lambda path: type(path) is int and session_2_mock or dir_mock
```

Comment:
What was the reason that this unit test for the ActiveSessions class need to change with the refactor?  I don't expect TestActiveSessions.py to change at all for this PR.